src/pages-and-resources/discussions/data/api.js

```diff
@@ -72,7 +72,7 @@ export function getApps() {
       },
     ],
     apps: [
-      edXForumsApp,
+      legacyEdXDiscussions,
```

Comment:
Does this imply we will later introduce a new, non-legacy `EdXDiscussions` app?

Depending on my assumptions, I have some concerns about this transition (when we actually do cut-over to "new" edx forums). My mental model roughly centered around us _not_ using the `DiscussionConfig` model for the legacy experience. In effect, the absence of _any_ entry in the new table meaning you get the old experience/logic everywhere, but if we have a row in this table, you get the new stuff (whether that's new edx or LTI, etc).

I have some particular thoughts/concerns about some use cases, that may be easier for me to talk through, given how many assumptions I'm making.

Reply:
It's worth noting that this is just test data for now; I don't think having two providers for legacy/new will mean we have to use DiscussionConfig for the legacy experience... I sort of assumed that in the short term the API endpoints would just delegate to the existing discussion config models and kinda do exactly what you said, i.e., not actually create a DiscussionConfig model for the legacy experience.